lemur/certificates/service.py

```diff
@@ -329,12 +329,14 @@ def render(args):
     """
     query = database.session_query(Certificate)
 
-    time_range = args.pop("time_range")
-    if not time_range:
-        six_month_old = arrow.now()\
-            .shift(months=current_app.config.get("HIDE_EXPIRED_CERTS_AFTER_MONTHS", -6))\
+    show_expired = args.pop("showExpired")
+    if show_expired != 1:
```

Comment:
This is contrary to how integers are interpreted as boolean everywhere else.

Usually a value `0` means False, everything else means True.

But this is implemented so that `2` is also effectively False.